packages/salesforce-adapter/test/adapter.crud.test.ts

```diff
@@ -1417,298 +1486,344 @@ describe('SalesforceAdapter CRUD', () => {
   describe('Update operation', () => {
     let result: DeployResult
     describe('for an instance element', () => {
-      let beforeInstance: InstanceElement
-      let afterInstance: InstanceElement
-
-      beforeEach(() => {
-        beforeInstance = createInstanceElement(mockDefaultValues.Profile, mockTypes.Profile)
-        afterInstance = createInstanceElement(
-          {
-            ...mockDefaultValues.Profile,
-            description: 'Updated profile description',
-          },
-          mockTypes.Profile,
-        )
-      })
-
-      describe('when the request succeeds', () => {
-        const DEPLOYMENT_ID = 'testDeploymentId'
+      describe('for non profile instances', () => {
```

Comment:
```suggestion
      describe('when the internal ID of the Element changes', () => {
```
We shouldn't specify implementation detail in the `describe` and `it`. We should describe the scenario we are testing and not the fact we invoke the code on non Profile instance. (This is simply an edge-case in the adapter that is not possible in Salesforce)